change/@ni-nimble-blazor-874611b4-f702-4410-ab94-6857594d58b8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could you see if there's a good place in the nimble-blazor CONTRIBUTING.md to add a note reminding us to do this for future components? 

I think the tests you added will also help remind us if we copy-paste code for new implementations, thanks for adding them!